src/applications/appeals/995/components/ContactInfo.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useEffect, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+import AddressView from '@@vap-svc/components/AddressField/AddressView';
+
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import { scrollAndFocus } from 'platform/utilities/ui';
+
+import { readableList } from '../utils/helpers';
+import { getFormatedPhone } from '../utils/contactInfo';
```

Comment:
getFormattedPhone*

Reply:
LOL that's a typo that I really don't want to fix in this PR 😛